mechafil_jax/locking.py

```diff
@@ -70,13 +76,15 @@ def compute_day_locked_pledge(
     baseline_power_eib: float,
     renewal_rate: float,
     scheduled_pledge_release: float,
-    lock_target: float = 0.3,
+    lock_target: float,
+    gamma: float,
+    gamma_weight_type: int, 
 ) -> float:
     # convert powers to PIB
     total_qa_power_pib = total_qa_power_eib * 1024.0
     baseline_power_pib = baseline_power_eib * 1024.0
 
-    # print('jax', day_network_reward, prev_circ_supply, day_onboarded_qa_power_pib, total_qa_power_pib, baseline_power_pib, lock_target)
+    print('jax', day_network_reward, prev_circ_supply, day_onboarded_qa_power_pib, total_qa_power_pib, baseline_power_pib, lock_target)
```

Comment:
print statements should be disabled before checking in, especially for JIT compiled code.
see here: 
https://jax.readthedocs.io/en/latest/jax-101/02-jitting.html#how-jax-transforms-work

Check the part where it talks about print statements not being "pure". 

Of course OK to have them in code while debugging though.